src/scf/self_consistent_field.jl

```diff
@@ -202,8 +206,7 @@ Overview of parameters:
 
     info_init = (; ρin=ρ, ψ=ψ, occupation=nothing, eigenvalues=nothing, εF=nothing, 
                    n_iter=0, n_matvec=0, timedout=false, converged=false,
-                   history_Etot=T[], history_Δρ=T[])
-
```

Comment:
Here you're killing an empty line, which I think would help the structure of the code.